client/header/activity-panel/activity-card/style.scss

```diff
@@ -287,3 +287,42 @@
 		}
 	}
 }
+
+.woocommerce-stock-activity-card {
+	.woocommerce-stock-activity-card__stock-quantity {
+		background: $core-grey-light-300;
+		color: $core-grey-dark-500;
+		padding: 3px $gap-smaller;
+		border-radius: 3px;
+	}
+
+	.woocommerce-stock-activity-card__edit-quantity {
+		display: inline-flex;
+		width: 50px;
+		margin-right: 10px;
+
+		input {
+			border-radius: 4px;
+			height: 36px;
+
+			&[type='number']::-webkit-outer-spin-button,
+			&[type='number']::-webkit-inner-spin-button {
+				-webkit-appearance: none;
+				margin: 0;
+			}
+		}
+	}
+
+	.is-button {
+		height: 34px;
```

Comment:
The _Cancel_ button looks a bit weird when clicked/focused. What would you think about making it higher or replacing the border with an underline?
![image](https://user-images.githubusercontent.com/3616980/55635238-82b8d780-57c0-11e9-87b0-767c6247a2d8.png)

Reply:
Maybe some padding such that it will be the same height as the other button.

Reply:
Fixed in bb445e3c33998fe32bdf730ae7f3be47c0203ddf.